draft-packages/button/KaizenDraft/Button/components/GenericButton.tsx

```diff
@@ -13,6 +13,16 @@ import React, {
 
 import styles from "./GenericButton.module.scss"
 
+// Duplicated(ish?) from design-tokens/layout
+type Breakpoints = {
+  "small": 0,
+  "medium": 768,
+  "large": 1080
+}
+
+type BreakpointNames = keyof Breakpoints
+type BreakpointMap<N> = N | { [key in BreakpointNames]: N }
```

Comment:
What's the best way to share these types across many packages?

Reply:
Probably makes sense to have these living in design-tokens somewhere.